content/javascript/concepts/npm/npm.md

```diff
@@ -53,3 +53,25 @@ Alternatively, run the following to display a (long) list of all NPM commands pa
 ```bash
 npm -l
 ```
+
+## Global Packages
+
+NPM also features the ability to install packages globally. This allows you to use packages as command line tools anywhere on your computer.
```

Comment:
```suggestion
npm also features the ability to install packages globally and use them as command line tools anywhere on the computer.
```

A few notes here:

- I think every instance of "NPM" should honestly be replaced with "npm." Even if it's the first word in the sentence. (per their [official site](https://docs.npmjs.com/about-npm))
- These two sentences could be consolidated into one.
- For Docs entries, we generally refrain from using a "you"/"your" voice in favor of a more neutral tone that is more manual and informative.

Reply:
Just checking, everywhere else on this page has NPM capitalized. Should I create a separate pull request to change this on the rest of the page?